src/management/list-items.factory.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+/**
+ * @ngdoc service
+ * @name udb.management.listItems
+ * @description
+ * # Management list items
+ * Return the management list items to show in the sidebar as  a promise.
+ */
+angular
+  .module('udb.management')
+  .factory('managementListItems', listItems);
```

Comment:
@kln3wrld Deze is verhuist van de app en heeft nog geen test-coverage. Opmerking iets lager over ontbrekende role constraints is al iets dat je in een spec kan gieten!